funding/manager.go

```diff
@@ -887,7 +925,7 @@ func (f *Manager) failFundingFlow(peer lnpeer.Peer, tempChanID [32]byte,
 	}
 
 	errMsg := &lnwire.Error{
-		ChanID: tempChanID,
+		ChanID: cid.tempChanID,
```

Comment:
I think there's at least one call-site where the _real_ ChannelID is used here, so it wouldn't use the tempChanID

Reply:
With the latest push this is now fixed.